ansible/env_setup.yml

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
Cant' we move this to a role instead of separate target

Reply:
Done!